### DISCUSSION TITLE
How to remove remote permission?

### DISCUSSION THREAD

**Discussion by kajal55409** [community/community#73446] [New to GitHub]
remote: Permission to kajal55409/cart.git denied to kajalguptaVE.
fatal: unable to access 'https://github.com/kajal55409/cart.git/': The requested URL returned error: 403

**Comment by RumethSandinu** [comment]
Try to clone again

**Reply by kajal55409** [reply]
I have tried many times.

**Comment by RumethSandinu** [comment]
You can remove remote permission on settings --> Collaborators

**Reply by kajal55409** [reply]
I am not able to find  settings --> Collaborators.
<img width="320" alt="Screenshot 2023-11-05 at 5 54 34 PM" src="https://github.com/community/community/assets/62256901/445889ac-beed-460b-a173-bf892989a4ef">
<img width="316" alt="Screenshot 2023-11-05 at 5 53 17 PM" src="https://github.com/community/community/assets/62256901/8fd44573-1666-4256-a5f4-4ccaf7621605">

**Comment by RumethSandinu** [comment]
No, you need to go to repository settings.
<img width="1338" alt="Screenshot 2023-11-05 at 6 13 22 PM" src="https://github.com/community/community/assets/137886587/024fc399-4faf-44e8-ac92-21657d8ca4aa">
<img width="1186" alt="Screenshot 2023-11-05 at 6 16 10 PM" src="https://github.com/community/community/assets/137886587/172113a3-7b20-49bc-a262-78cbe6e21196">

**Reply by kajal55409** [reply]
<img width="1465" alt="Screenshot 2023-11-05 at 9 53 07 PM" src="https://github.com/community/community/assets/62256901/f994a30b-6140-433f-9151-8b93d902c0c2">

**Reply by kajal55409** [reply]
didn't find the remote section here.

**Reply by RumethSandinu** [reply]
Do you need to delete the repository?

**Reply by kajal55409** [reply]
My error is 

remote: Permission to kajal55409/reduxCart.git denied to kajalguptaVE.
fatal: 

I am not able to push a code on my git hub, it's showing remote permission denied.

**Reply by RumethSandinu** [reply]
I got it now. Download github for desktop and sign in to it, then try to push using that or you can try to push using any jetbrain IDE.

**Reply by kajal55409** [reply]
okay I'll try this also.

**Reply by kajal55409** [reply]
It will work on Macos ?

I have searched App store all i get is git hub wiki search.
<img width="1470" alt="Screenshot 2023-11-06 at 5 21 10 AM" src="https://github.com/community/community/assets/62256901/545448df-524c-48a7-9df9-7bf6a93d22fa">

**Reply by RumethSandinu** [reply]
I am also using macos. It is not in the app store. Search for github for desktop or go to this link and install.
https://desktop.github.com

**Comment by RumethSandinu** [comment]
I recommend you to watch this tutorials
https://www.youtube.com/watch?v=3RjQznt-8kE&list=PL4cUxeGkcC9goXbgTDQ0n_4TBzOO0ocPR

**Reply by kajal55409** [reply]
Thank you so much. I'll definitely watch this.

**Comment by airtower-luna** [comment]
> remote: Permission to kajal55409/cart.git denied to kajalguptaVE.

Mind the different user names! Maybe you have credentials for the wrong user stored somewhere?

**Reply by kajal55409** [reply]
It seems I have stored it, but how do get remove that?

**Reply by airtower-luna** [reply]
That depends on the kind of credential storage you're using. You mentioned using macOS above, so *probably* its keychain. I'm not familiar with macOS, so I can't say how to edit that.

If you want to check, see the output of `git config --get credential.helper`. See [Git Tools - Credential Storage](https://git-scm.com/book/en/v2/Git-Tools-Credential-Storage) for what it means.
